git_machete/cli.py

```diff
@@ -475,9 +475,9 @@ def set_utils_global_variables(
     utils.verbose_mode = cli_opts.opt_verbose
 
 
-def get_branch_arg_or_current_branch(
-        cli_opts: git_machete.options.CommandLineOptions, git_context: GitContext) -> LocalBranchShortName:
-    return cli_opts.opt_branch or git_context.get_current_branch()
+def get_local_branch_short_name_from_arg_or_current_branch(
+        branch_from_arg: Optional[AnyBranchName], git_context: GitContext) -> LocalBranchShortName:
+    return LocalBranchShortName.of(branch_from_arg.replace('refs/heads/', '')) if branch_from_arg else None or git_context.get_current_branch()
```

Comment:
🤔 
Maybe just:
```suggestion
    return LocalBranchShortName.of(branch_from_arg.replace('refs/heads/', '')) if branch_from_arg else git_context.get_current_branch()
```
❓